### PR TITLE
Adds Codesandbox links to components!

### PR DESCRIPTION
**Pull request by bruugey**
- Updates existing logo icons
- Fixes annoying border

### CODE REVIEW

File: layouts/ComponentLayout.tsx [hunk 1]
```diff
@@ -78,10 +77,16 @@ const desktopLinksContainer = css`
 const tabStyles = css`
   width: 100%;
   max-width: 100%;
+
```
Comment (shaneeza) [inline layouts/ComponentLayout.tsx:80]: I think sean fixed the uneven border in the tabs component, are we using the latest?

Comment (shaneeza) [reply layouts/ComponentLayout.tsx:80]: https://github.com/mongodb/leafygreen-ui/pull/2040

Comment (bruugey) [reply layouts/ComponentLayout.tsx:80]: is staging outof sync from main?

Comment (shaneeza) [reply layouts/ComponentLayout.tsx:80]: `Tabs` is at `v11.1.13` which is correct, maybe the fix didn't work?

File: utils/ContentStack/types.ts [hunk 2]
```diff
@@ -31,6 +31,7 @@ export interface ComponentPageMeta {
   url: string;
   description: string;
   figmaurl?: string;
+  codesandbox_url?: { title?: string; href?: string };
```
Comment (shaneeza) [inline utils/ContentStack/types.ts:34]: whats the `title` for?

Comment (bruugey) [reply utils/ContentStack/types.ts:34]: it's just how the data is stored in contentstack

Comment (TheSonOfThomp) [reply utils/ContentStack/types.ts:34]: Can we change that on the CS side to match the figmaurl schema?

Comment (bruugey) [reply utils/ContentStack/types.ts:34]: Figma url was aded as a text field, this is technically more correct - but I can update for consistency if you all prefer.

File: layouts/ComponentLayout.tsx [hunk 3]
```diff
@@ -108,6 +123,7 @@ const ComponentLinks = ({
     </IconButton>
     {component.figmaurl && (
       <IconButton
+        key="figma"
```
Comment (TheSonOfThomp) [inline layouts/ComponentLayout.tsx:126]: do we need a `key` in an un-mapped element?

Comment (bruugey) [reply layouts/ComponentLayout.tsx:126]: was requesting one

**Review by TheSonOfThomp — APPROVED** [review]

**Pull request merged**